Simulator/Connections/NG911/Connections911.cpp

```diff
@@ -43,7 +43,7 @@ void Connections911::setup()
       edgeType type = layout->edgType(srcV, destV);
       BGFLOAT *sumPoint = &vertices->summationMap_[destV];
 
-      BGFLOAT dist = (*layout->dist_)(srcV, destV);
+      BGFLOAT dist = (layout->dist_)(srcV, destV);
```

Comment:
I believe the parentheses around `(layout->dist_)` are no longer necessary.

Reply:
Will fix this, thanks!